### PR TITLE
RDSC-1367-replace-all-download-link-in-preview-docs-with-a-var

### PR DESCRIPTION
**Pull request by yaronp68**

### CODE REVIEW

**Review by spicy-sauce — CHANGES_REQUESTED** [review]

File: config.toml [hunk 1]
```diff
@@ -54,6 +54,8 @@ rdi_redis_gears_current_semantic_version = "1.2.6"
 rdi_redis_gears_current_version = "10206"
 rdi_debezium_server_version = "2.1.1.Final"
 rdi_db_types = "cassandra|mysql|oracle|postgresql|sqlserver"
+rdi_cli_latest= "0.106.0"
```
[the remark this answers is not in the document]
Comment (yaronp68) [reply config.toml:57]: @spicy-sauce please put the build for latest we want to use

[the remark this answers is not in the document]
Comment (spicy-sauce) [reply config.toml:57]: 0.107.0

File: config.toml [hunk 2]
```diff
@@ -54,6 +54,8 @@ rdi_redis_gears_current_semantic_version = "1.2.6"
 rdi_redis_gears_current_version = "10206"
 rdi_debezium_server_version = "2.1.1.Final"
 rdi_db_types = "cassandra|mysql|oracle|postgresql|sqlserver"
+rdi_cli_latest = "0.107.0"
```
Comment (spicy-sauce) [inline config.toml:57]: You don't use it, only the `rdi_cli_preview`, so is it needed?

**Review by spicy-sauce — COMMENTED** [review]
BTW, in `debezium-server-deployment.md`, we have this:
```
  tar xvfz debezium-server-dist-{{ <param rdi_debezium_server_version> }}.tar.gz
```

While in all other files, the placeholders are specified like this:
```
  tar xvfz debezium-server-dist-{{<param rdi_debezium_server_version>}}.tar.gz
```

This is why the placeholder not replaced here:
https://docs.redis.com/latest/rdi/installation/debezium-server-deployment/

Updated it in this PR.

File: config.toml (hunk 2 above)
Comment (yaronp68) [reply config.toml:57]: I want to be able to move easy into master when we release

**Pull request merged**